src/components/Auth/ChangePassword/ChangePassword.react.js

```diff
@@ -277,10 +277,17 @@ export default class ChangePassword extends Component {
     };
     const labelStyle = {
       width: '30%',
+      minWidth: '150px',
       float: 'left',
       marginTop: '12px',
       color: UserPreferencesStore.getTheme() === 'light' ? 'black' : 'white',
     };
+    const submitBtn = {
```

Comment:
Can you change the name to `submitBtnStyle` to indicate it's a style object? Rest looks fine.